.github/workflows/publish-headless-example.yml

```diff
@@ -1,111 +0,0 @@
-name: Publish headless-example image
```

Comment:
Is there a place / github workflow that calls this script?

Reply:
There are no references to this file from other files.